client/src/domain/Auth/Login/index.js

```diff
@@ -1,7 +1,64 @@
 import './style.scss';
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
 
 function Login() {
-  return <div className="login">Rendering Login in here</div>;
+  const [formData, setFormData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const { email, password } = formData;
+
+  const onChange = (event) => {
+    setFormData({ ...formData, [event.target.name]: event.target.value });
+    console.log(email, password);
+  };
+
+  const onSubmit = (event) => {
+    event.preventDefault();
+    console.log(email, password);
+  };
+
+  // TODO: add context to retrieve info if the user is already authenticated
+  // if (isAuthenticated) {
+  //   return <Redirect to="/user/me" />;
+  // }
+
+  return (
+    <div className="auth">
+      <h1>Login</h1>
+      <form className="auth__form" onSubmit={onSubmit}>
+        <div className="auth__form-group">
+          <input
+            type="email"
+            placeholder="Email Address"
+            name="email"
+            value={email}
+            onChange={onChange}
+            required
+          />
+        </div>
+        <div className="auth__formgroup">
+          <input
+            type="password"
+            placeholder="Password"
+            name="password"
+            value={password}
+            onChange={onChange}
+            minLength="5"
+          />
+        </div>
+        {/* TODO: add submit logic */}
+        <button type="submit" className="button--submit">
+          Login
+        </button>
+      </form>
```

Comment:
Consider using a separate package for controlled inputs. E.g. Formik or React useForm Hook. You can also use yup for validation schemas.

Reply:
Thanks, I'll check it out!